pennylane/devices/default_gaussian.py

```diff
@@ -689,6 +690,7 @@ def __init__(self, wires, *, shots=None, hbar=2, analytic=None):
         super().__init__(wires, shots, analytic=analytic)
         self.eng = None
         self.hbar = hbar
+        self.debugger = None
```

Comment:
Should the debugger be reset in `dev.reset`? If not, how could it be flushed?

Reply:
Yes we could add a reset. Right now it's not required because the debugger is not public, and is only created in the `qml.snapshots` function where it is discarded again after use.

In future releases where we potentially implement a more advanced debugger this could be relevant.